src/App.js

```diff
@@ -9,18 +10,7 @@ import { goToLogIn, registrationError } from "./ducks";
 
 class App extends Component {
 	componentDidMount() {
-		// firebase.auth().signOut();
-
-		const config = {
```

Comment:
This config shouldn't be removed. It should be replaced with getting the info from .env files and just don't push the .env files